api/model.ts

```diff
@@ -2,4 +2,23 @@ export type Snapshot = {
     checkpointBlocks: string[]
     hash: string
     height: number
+    timestamp: string
```

Comment:
Consider keeping timestamp as `Date` but for this you will need some kind of mapper between elasticsearch and internal DTO as well as from internal DTO to json output (so both in/out)